libs/@local/hash-validation/src/property_type.rs

```diff
@@ -101,22 +103,6 @@ where
     }
 }
 
-impl<P> Validate<PropertyType, P> for PropertyWithMetadata
```

Comment:
why the removal of validate? I get it is just redirecting to validate value.

Reply:
Yes, it's just a redirection which bloats code. Not needed.